.travis.yml

```diff
@@ -7,14 +7,18 @@ cache:
         - $HOME/.composer/cache
 
 php:
-  - '5.5'
-  - '5.6'
-  - '7.0'
-  - hhvm
+  - '7.3'
+  - '7.4'
+  - '8.0'
   - nightly
 
+before_install:
+  - sudo apt-get update
+  - sudo apt-get install -y ant
```

Comment:
The ant requirement should probably just be listed in the README. This will fail on systems that are not based on debian.

Reply:
This is only used for our own code checking to guard the quality of our code.